sql/src/test/java/io/crate/analyze/AlterTableRerouteAnalyzerTest.java

```diff
@@ -73,7 +73,7 @@ public void testRerouteMoveShardPartitionedTable() throws Exception {
 
     @Test
     public void testRerouteOnBlobTable() throws Exception {
-        RerouteMoveShardAnalyzedStatement analyzed = e.analyze("ALTER TABLE blob.blobs REROUTE MOVE SHARD 0 FROM 'nodeOne' TO 'nodeTwo'");
```

Comment:
We should probably keep a test for the `ALTER TABLE blob.blobs` to not cause a regression in the future.